app/models/consumer_role.rb

```diff
@@ -1287,4 +1285,10 @@ def verification_attr(*authority)
                    })
   end
 
+  def determine_verifications_event
```

Comment:
Can you rename this method name to something more generic name like **publish_created_event** or **notify_created** as it is just publishing a ConsumerRoleCreated event?